src/style.css

```diff
@@ -2,0 +1,90 @@
+body {
+  font-family: ' Avenir', Arial, sans-serif;
+}
+
+input {
+  all: unset;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+}
+
+@font-face {
+  font-family: ' Avenir', sans-serif;
```

Comment:
The font-family name 'Avenir' in the @font-face rule should match exactly with the font-family usage in the rest of your CSS to avoid any potential issues with font loading.